net-vpn/iodine/iodine-0.7.0-r2.ebuild

```diff
@@ -21,9 +21,13 @@ RDEPEND="sys-libs/zlib"
 DEPEND="${RDEPEND}
 	test? ( dev-libs/check )"
 
+PATCHES=(
+	"${FILESDIR}"/${P}-TestMessage.patch
+	"${FILESDIR}"/${P}-new-systemd.patch
+)
+
 src_prepare() {
-	epatch "${FILESDIR}"/${P}-TestMessage.patch
-	epatch "${FILESDIR}"/${P}-new-systemd.patch
+	default
 
 	sed -e '/^\s@echo \(CC\|LD\)/d' \
 		-e 's:^\(\s\)@:\1:' \
```

Comment:
re src_compile (wrong line, I can't comment on the correct one):
- "# Shipped ..."
- check Makefile for possible values of TARGETOS and use os_* or kernel_* use flags (they're implicit) to provide correct values.

Reply:
I wonder if even need that line, because invoking `emake` in the parent directory (not in ./src) works fine, and the top makefile sets TARGETOS using `uname`

Reply:
Yeah, let's drop it.